playbooks/roles/ocp-odf-operator/tasks/main.yml

```
@@ -0,0 +1,330 @@
+      spec:
+        nodeSelector:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: kubernetes.io/hostname
+              operator: In
+              values:
+              - "{{ worker0_name }}"
+              - "{{ worker1_name }}"
+              - "{{ worker2_name }}"
+  when: lvdisc_count.stdout|int < 1
+
+- name : Verify LocalVolumeDiscovery instance is created
+  shell:  oc get LocalVolumeDiscovery -n openshift-local-storage | wc -l
+  register: lvd_cr
+  until: lvd_cr.stdout|int == 2
+  retries: 10
+  delay: 120
+
+- name: Verify LVD pods are created
+  shell: oc get pod -n openshift-local-storage --selector app=diskmaker-discovery | wc -l
+  register: pods_count
+  until: pods_count.stdout|int > 3
+  retries: 10
+  delay: 60
+
+
+# Create the StorageCluster CR
+- name: Create the StorageCluster Custom Resource
+  k8s:
+    state: present
+    definition:
+      apiVersion: local.storage.openshift.io/v1alpha1
+      apiVersion: ocs.openshift.io/v1
+      kind: StorageCluster
+      metadata:
+        annotations:
+          cluster.ocs.openshift.io/local-devices: 'true'
+        name: ocs-storagecluster
+        namespace: openshift-storage
+      spec:
+        flexibleScaling: true
+        manageNodes: false
+        monDataDirHostPath: /var/lib/rook
+        storageDeviceSets:
+        - count: 3
+          dataPVCTemplate:
+            spec:
+              accessModes:
+              - ReadWriteOnce
+              resources:
+                requests:
+                  storage: 100Gi
+              storageClassName: localblock
+              volumeMode: Block
+          name: ocs-deviceset-localblock
+          placement: {}
+          portable: false
+          replica: 1
+          resources: {}
+
+# To verify that the Operator deployment is successful
+- name: Verify the StorageCluster instance is in Ready state
+  shell: oc get storagecluster ocs-storagecluster -n openshift-storage -ojsonpath={.status.phase}
+  register: sc_status
+  until: sc_status.stdout|string == "Ready"
+  retries: 10
+  delay: 180
+
+- name: Verify the CephCluster health is OK
+  shell: oc get cephcluster ocs-storagecluster-cephcluster -n openshift-storage -ojsonpath={.status.phase}
+  register: cc_status
+  until: cc_status.stdout|string == "Ready"
+  retries: 10
+  delay: 180
+
+- name: Verify the Storage Classes are created
+  shell: oc get sc  --output custom-columns="PROVISIONER:provisioner" | grep openshift-storage | wc -l
+  register: sc_count
+  until: sc_count.stdout|int == 4
+  retries: 10
+  delay: 120
+
+# Validate ODF deployment
+- name: Validate File storage
+  block:
+  - name: Create file storage yaml
+    template:
+      src: "{{ role_path }}/templates/file_storage.yaml.j2"
+      dest: "{{ role_path }}/files/file_storage.yml"
+
+  - name: Create POD with PVC using ocs-storagecluster-cephfs storageclass
+    shell: oc apply -f "{{ role_path }}/files/file_storage.yml"
+
+  - name: Check that Pod is running
```

Can you please re-name the task?  
```suggestion
  - name: Check file storage test pod is running
```

---

Done